cmd/main.go

```diff
@@ -1,5 +1,21 @@
 package main
 
+import (
+	"log"
+	"os"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/server"
+)
+
 func main() {
+	logger := log.New(os.Stdout, "MorseAppConverter: ", log.LstdFlags)
+
+	servak := server.NewServer(logger)
+
+	logger.Println("Start server...")
 
+	err := servak.HTTPServer.ListenAndServe()
+	if err != nil {
+		logger.Fatal()
```

Comment:
У тебя есть ошибка, но ты ее не выводишь, поэтому у меня вот проблема с запуском, но я не знаю причину
![image](https://github.com/user-attachments/assets/a1a14ef0-ccb2-4f14-ac0b-a93902d5a898)
```suggestion
		logger.Fatal(err)
```